kubernetes-maven-plugin/doc/src/main/asciidoc/inc/goals/build/_jkube-helm.adoc

```diff
@@ -24,57 +28,101 @@ The configuration happens in a `<helm>` section within the plugin's configuratio
+| `jkube.helm.description`
 
-| *outputDir*
-| Where to create the the Helm chart, which is `${basedir}/target/jkube.helm` by default for Kubernetes (and `${basedir}/target/jkube.helmshift` for OpenShift).
-| `jkube.helm.outputDir`
+| *home*
+| The Chart URL for this project's home page, which is `${project.url}` if not provided.
+| `jkube.helm.home`
+
+| *sources*
+| The Chart list of URLs to source code for this project, defaults to the list of `${project.scm.url}` if not provided.
+|
+
+| *maintainers*
+| The Chart list of maintainers (name+email), defaults to the list of
+  `${project.developers.name}:${project.developers.name}`if not provided.
+|
+
+| *icon*
+| The Chart URL to an SVG or PNG image to be used as an icon, default is extracted from the kubernetes manifest
+  (`kubernetes.yml`) `jkube.io/iconUrl` annotation if not provided.
+| `jkube.helm.icon`
 
 | *keywords*
-| Comma separated list of keywords to add to the chart
+| Comma separated list of keywords to add to the chart.
 |
 
 | *engine*
-| The template engine to use
+| The template engine to use.
+|
+
+| *additionalFiles*
+| The list of additional files to be included in the Chart archive. Any file named `README` or `LICENSE` will *always*
+  be included by default.
 |
 
+| *type* / *types*
+| Platform for which to generate the chart. By default this is `kubernetes`, but can be also `openshift` for using
+  OpenShift specific resources in the chart. You can also add both values as a comma separated list.
+
+  _Please note that there is no OpenShift support yet for charts, so this is experimental._
+| `jkube.helm.type`
+
+| *sourceDir*
+| Where to find the resource descriptors generated with `k8s:resource`.
+  By default this is `${basedir}/target/classes/META-INF/jkube., which is also the output directory used by `k8s:resource`.
+| `jkube.helm.sourceDir`
+
+| *outputDir*
+| Where to create the the Helm chart, which is `${basedir}/target/jkube/helm` by default for Kubernetes
+  and `${basedir}/target/jkube/helmshift` for OpenShift.
+| `jkube.helm.outputDir`
+
+| *tarballOutputDir*
+| Where to create the the Helm chart archive, which is `${basedir}/target` if not provided.
+| `jkube.helm.tarballOutputDir`
+
 | *chartExtension*
-| The Helm chart file extension, default value is `tar.gz`
+| The Helm chart file extension (`tgz`, `tar.bz`, `tar.bzip2`, `tar.bz2`), default value is `tar.gz` if not provided.
 | `jkube.helm.chartExtension`
 |
 |===
 
 
-In a next step you can install this via the https://github.com/kubernetes/helm/releases[helm command line tool] as follows:
+In a next step you can install this via the https://github.com/helm/helm/releases[helm command line tool] as follows:
 
 [source, sh]
 ----
-helm install target/jkube.helm/kubernetes
+helm install nameForChartInRepository target/jkube.helm/kubernetes
+----
+or
+----
+helm install target/jkube.helm/kubernetes --generate-name
```

Comment:
Is directory name really `jkube.helm`? Or `jkube/helm`?

Reply:
jkube.helm was the old one, it's jkube/helm, forgot to change it here.